src/Core/IIssuesFilter.cs

```diff
@@ -19,12 +19,25 @@
  */
 
 using System.Collections.Generic;
-using Sonarlint;
 
-namespace SonarLint.VisualStudio.Integration.Vsix.SonarLintTagger
+namespace SonarLint.VisualStudio.Core
 {
+    /// <summary>
+    /// Describes a single issue with the properties required for
+    /// it to be compared against server-side issues by the issues filter
+    /// </summary>
+    public interface IFilterableIssue
```

Comment:
Maybe extract this to another file?

Reply:
The interfaces are related and both very short; I didn't see any upside in having them in separate file.